types/src/block_info.rs

```diff
@@ -174,3 +175,43 @@ impl Display for BlockInfo {
         )
     }
 }
+
+#[cfg(any(test, feature = "fuzzing"))]
+impl Arbitrary for BlockInfo {
+    type Parameters = ();
+    type Strategy = BoxedStrategy<Self>;
+
+    fn arbitrary_with(_args: Self::Parameters) -> Self::Strategy {
+        (
+            any::<u64>(),
+            any::<Round>(),
+            any::<HashValue>(),
+            any::<HashValue>(),
+            any::<Version>(),
+            any::<u64>(),
+            prop_oneof![Just(None), any::<EpochState>().prop_map(Some)],
```

Comment:
tbh I don't know what would happen with random BlockInfo in LedgerInfo, there's assumption that if a LedgerInfoWithSignature has valid 2f+1 signatures, the content must be valid.